<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParentDialogClass</class>
 <widget class="QDialog" name="ParentDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ParentDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QLineEdit" name="email_edit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Email...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="gender_label">
     <property name="text">
      <string>Gender:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="children_label">
     <property name="text">
      <string>Children:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name_edit">
     <property name="placeholderText">
      <string>Name...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="surname_label">
     <property name="text">
      <string>Surname:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="email_label">
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="name_label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QScrollArea" name="children">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="children_contents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>194</width>
        <height>85</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="surname_edit">
     <property name="placeholderText">
      <string>Surname...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="gender_comboBox">
     <item>
      <property name="text">
       <string>Mother</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Father</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ParentDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ParentDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
